Tensile/KernelWriterAssembly.py

```diff
@@ -12013,6 +12020,11 @@ def globalWriteBatch(self, kernel, ss, batchIdx, applyAlpha, beta, edge, atomic,
             # CEqualsD case, use SrdD instead of SrdC
             LoadCCodeStr = LoadCCodeStr.replace('SrdC', 'SrdD')
           LoadCCodeMod.addCode(LoadCCodeStr)
+          
+      if not kernel["BufferStore"]:
+        kStr += addrCalc.emitAddressSetupCode(kernel, ss, 'D', tmpVgpr, tmpS01, edge, beta, atomic, elementIdx, addrDVgpr)
+        if edge:
```

Comment:
We probably do not need to call edgeProtectCode twice for (not BufferStore) + edge + beta case.
It seems not making any difference.